plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/product-edit-block-editor.spec.js

```diff
@@ -80,3 +80,84 @@ test( 'can update the general information of a product', async ( {
 			.toHaveText( updatedProduct.description );
 	} );
 } );
+
+test.describe( 'Publish dropdown options', () => {
+	test( 'can schedule a product publication', async ( { page, product } ) => {
+		await page.goto( `wp-admin/post.php?post=${ product.id }&action=edit` );
+
+		await page
+			.locator( '.woocommerce-product-header__actions' )
+			.first()
+			.locator( 'button[aria-label="More options"]' )
+			.click();
+
+		await page.getByText( 'Schedule publish' ).click();
+
+		await expect(
+			page.getByRole( 'heading', { name: 'Schedule product' } )
+		).toBeVisible();
+
+		await page
+			.locator( '.woocommerce-schedule-publish-modal' )
+			.locator( 'button[aria-label="View next month"]' )
+			.click();
+
+		await page
+			.locator( '.woocommerce-schedule-publish-modal' )
+			.getByText( '14' )
+			.click();
+
+		await page.getByRole( 'button', { name: 'Schedule' } ).click();
+
+		await expect(
+			page.getByLabel( 'Dismiss this notice' ).first()
+		).toContainText( 'Product scheduled for' );
+	} );
+	test( 'can dupicate a product', async ( { page, product } ) => {
```

Comment:
```suggestion
	test( 'can duplicate a product', async ( { page, product } ) => {
```
Small spelling mistake